bin/citrea/tests/e2e/sequencer_behaviour.rs

```diff
@@ -577,10 +570,41 @@ async fn test_sequencer_fills_empty_blocks_for_missed_da_blocks() -> Result<(),
 
     let seq_test_client = make_test_client(seq_port).await?;
 
-    // Invoke the loop / select in sequencer to process missed DA blocks.
-    seq_test_client.send_publish_batch_request().await;
-    // The sequencer should create an empty block per missed finalized DA block.
-    wait_for_l2_block(&seq_test_client, 13, None).await;
+    // will produce some blocks
+    // should also produce for the missing DA blocks
+    tokio::time::sleep(Duration::from_secs(3)).await;
+
+    let mut last_used_l1_height = 3;
+
+    let head_soft_confirmation_height = seq_test_client
+        .ledger_get_head_soft_confirmation_height()
+        .await
+        .unwrap()
+        .unwrap();
+
+    // check that the sequencer has at least one block for each DA block
+    // starting from DA #3 all the way up to DA #13 without no gaps
+    // the first soft confirmation should be on DA #3
+    // the last soft confirmation should be on DA #13
+    for i in 1..=head_soft_confirmation_height {
```

Comment:
How about start from 2? You won't need the if statement below.

Reply:
I wanted to guarantee sequencer started block production from the last finalized L1 height it first saw on startup.

so if we break it in the future this test will detect it 

like a sanity check let's say